Marlin/src/gcode/bedlevel/G26.cpp

```diff
@@ -159,28 +162,27 @@ int8_t g26_prime_flag;
    * Detect is_lcd_clicked, debounce it, and return true for cancel
    */
   bool user_canceled() {
-    if (!is_lcd_clicked()) return false;
-    safe_delay(10);                       // Wait for click to settle
+    if (!is_lcd_clicked()) return false; // Return if the button isn't pressed
 
     #if ENABLED(ULTRA_LCD)
       lcd_setstatusPGM(PSTR("Mesh Validation Stopped."), 99);
       lcd_quick_feedback();
     #endif
 
-    while (!is_lcd_clicked()) idle();    // Wait for button release
+    safe_delay(10);                      // Wait for click to settle
+    while (!is_lcd_clicked()) idle();    // Wait for button press again?
```

Comment:
@Roxy-3D — This code is actually waiting for a press. Should it be changed to `wait_for_release()` instead?